micrometer-core/src/test/java/io/micrometer/core/instrument/binder/jvm/JvmInfoMetricsTest.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 VMware, Inc.
```

Comment:
2021? :)

Reply:
I feel like I wrote that .. a long time ago.. LOL